sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusProcessorTest.java

```diff
@@ -101,63 +94,19 @@ public void testReceivingMessagesWithProcessor(boolean isV2) throws InterruptedE
         assertTrue(success, "Failed to receive all expected messages");
     }
 
-    /**
-     * Tests receiving messages using a session-enabled {@link ServiceBusProcessorClient}.
-     *
-     * @throws InterruptedException If the test is interrupted.
-     */
-    @Test
-    public void testReceivingMultiSessionMessagesWithProcessor() throws InterruptedException {
```

Comment:
The `ServiceBusSessionReceiverAsyncClient::ServiceBusReceiverAsyncClient` is no longer responsible for internally pumping from "multiple" sessions, the v2 rework move this responsibility to `SessionsMessagePump` to address the out of order message delivery and the limited concurrency v1 had.  The same case is now covered by 
`SessionsMessagePumpIsolatedTest::shouldPumpFromMultiSessionWhenBackingProcessor`